CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+## 0.10.0 / 2017-11-09
+* [ENHANCEMENT] Improve amtool check-config use and description text (#1016)
+* [ENHANCEMENT] Add metrics about current silences and alerts (#998)
+* [ENHANCEMENT] Sorted silences based on current status (#1015)
+* [ENHANCEMENT] add metric of alertmanager position in mesh (#1024)
+* [ENHANCEMENT] Initialise notifications_total and notifications_failed_total (#1011)
+* [ENHANCEMENT] Allow selectable matchers on silence view (#1030)
+* [ENHANCEMENT] Rename Promtheus to Alertmanager in CONTRIBUTING.md (#1033)
+* [ENHANCEMENT] Allow template in victorops message_type field (#1038)
+* [ENHANCEMENT] Optionally hide inhibited alerts in API response (#1039)
+* [ENHANCEMENT] Improve config test coverage (#1046)
+* [ENHANCEMENT] Convert Alertmanager to use non-global go-kit loggers (#1052)
+* [ENHANCEMENT] Add tests to notifiers config (#1050)
+* [ENHANCEMENT] Toggle silenced and inhibited alerts in UI (#1049)
+* [ENHANCEMENT] Fix pushover limits (title, message, url) (#1055)
+* [ENHANCEMENT] Add limit to OpsGenie message (#1045)
+* [ENHANCEMENT] Add logging to statemessage truncation (#1056)
+* [ENHANCEMENT] Upgrade OpsGenie notifier to v2 API. (#1061)
+* [ENHANCEMENT] Fix documentation (#1065)
+* [ENHANCEMENT] Allow enabling mutex and block profiles (#1073)
+* [ENHANCEMENT] Allow template in victorops routing_key field (#1083)
+* [ENHANCEMENT] Add support for PagerDuty API v2 (#1054)
+* [ENHANCEMENT] Prevent alerts from inhibiting themselves (#1017)
```

Comment:
This is a CHANGE, and more accurately it prevents inhibiting alerts in the source of the inhibition.

Reply:
@stuartnelson3